vspace/src/main/java/edu/asu/diging/vspace/core/services/impl/SpaceManager.java

```diff
@@ -179,6 +180,13 @@ public ISpace getFullyLoadedSpace(String id) {
         return spaces;
     }
 
+    @Override
+    public List<ISpace> getPublishedSpaces(SpaceStatus status) {
```

Comment:
since this method takes a status as parameter and will return all spaces with that status (not necessarily published), the method should have a different name that reflects that

Reply:
needs test cases